org.unitystation.StationHub.yaml

```diff
@@ -0,0 +1,71 @@
+app-id: org.unitystation.StationHub
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: stationhub.sh
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=all
+  - --share=network
+  - --filesystem=~/.var/app/org.unitystation.StationHub:create
```

Comment:
This should technically happen automatically on install/launch. I don't think you need it.